src/test/java/core/basesyntax/model/FruitTransactionTest.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax.model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class FruitTransactionTest {
```

Comment:
we don't test models, they are considered to be tested indirectly via services that use them